sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/AnyValue.scala

```diff
@@ -32,7 +32,8 @@ import org.apache.spark.sql.types._
 @ExpressionDescription(
   usage = """
     _FUNC_(expr[, isIgnoreNull]) - Returns some value of `expr` for a group of rows.
-      If `isIgnoreNull` is true, returns only non-null values.""",
+      If `isIgnoreNull` is true, returns only non-null values.
+  """,
```

Comment:
change to use `"`? I found other places have used `"` instead of `"""`

Reply:
Yeah. So we need to simplify and unify them.

Reply:
`"""` brings "\n" after `a group of rows.`